### PR TITLE
User suggested songs

### PR DESCRIPTION
**Pull request by shknbk**
This includes the profile page with basic styling and user song functionality.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -44,29 +44,18 @@ def current_user
     end
   end
 
+  get"/profile" do
+    @user_songs = current_user.user_songs
```
Comment (jamesdabbs) [inline app.rb]: What if we're not logged in?

File: lib/user.rb [hunk 2]
```diff
@@ -1,4 +1,14 @@
 class User < ActiveRecord::Base
   validates_presence_of :email, :password
   validates_uniqueness_of :email
+
+  def user_songs
+    user_song_arr = []
+    Song.all.each do |song|
+      if song.suggested_by.id == self.id
```
Comment (jamesdabbs) [inline lib/user.rb]: Can we replace this with `.suggested_by_id` instead of `.suggested_by.id` and save query?

File: seeds.rb [hunk 3]
```diff
@@ -24,12 +24,12 @@
...
+  artist: "The Weekend"},
   {
   title: "Locked Away",
-  artist: "R. City Featuring Adam Levine"}
+  artist: "R. City Featuring Adam Levine"},
   ]
 
 songs.each do |song|
-  Song.create! title: song[:title], artist: song[:artist], suggested_by: User.all.sample 
+  Song.create! title: song[:title], artist: song[:artist], suggested_by: User.find_by(email: "devin@example.com")
```
Comment (jamesdabbs) [inline seeds.rb]: I think we probably want to leave this random. You can always change them in your own console if you want.

Comment (andrewratliff) [inline seeds.rb]: Don't need to make any of these changes to seed.rb

**Comment by jamesdabbs** [comment]
:+1:

**Comment by jamesdabbs** [comment]
:shipit:

**Comment by jamesdabbs** [comment]
:ship:

**Pull request merged**